### ISSUE TITLE
how to distinguish three different methylation types in DMR?

### ISSUE THREAD

**Issue by entelechy001** [guoweilong/cgmaptools#69]
I’m running CGmaptools to get DMR from my WGBS data. In my input CGMAP format, there are three types methylation, CG, CHG, CHH. But the output format does not distinguish between these three methylation types. So is there a way we can distinguish between these three types?

Thank you for your help.

**Comment by Miaolingfeng** [comment]
No, create separate files for each type and then run the DMR program.
